File: 1207-unique-number-of-occurrences/1207-unique-number-of-occurrences.cpp
class Solution {
public:
    bool uniqueOccurrences(vector<int>& arr) {
        int N=2001;
        int count[2001]={0};
        for(int i=0; i<arr.size(); i++){
            count[arr[i] + 1000]++;
        }
        
        sort(count, count+N);
        for(int i=1; i<N-1; i++){
            if(count[i] > 0 && (count[i] == count[i+1])) return false;
        }
        
        return true;
    }
};

File: 1207-unique-number-of-occurrences/1207-unique-number-of-occurrences.py
class Solution:
    def uniqueOccurrences(self, arr: List[int]) -> bool:
        
        count = [0]*2001
        for num in arr:
            count[num + 1000] += 1
        
        count.sort()
        for i in range(1, len(count)-1):
            if count[i]>0 and count[i] == count[i+1]:
                return False
        return True
    